state/state_test.go

```diff
@@ -958,6 +962,7 @@ func TestExecutorRevert(t *testing.T) {
 //		Db:               genesisDB,
 //		ChainId:          stateCfg.ChainID,
 //		ForkId:           forkID,
+//		ContextId:        uuid.NewString(),
```

Comment:
Why this test is commented?

Reply:
I don't know, I just added the new field in case we uncomment this in the future.